playbooks/openshift/heat_deprovision.yml

```diff
@@ -12,14 +12,15 @@
         server: "{{ cluster_name }}-lb-0"
         floating_ip_address: "{{ openshift_public_ip }}"
         state: absent
-      when: lb_vm_group_size > 0
+      when: master_vm_group_size > 1
```

Comment:
I can see the logic here (no lb_vm_group_size for single master deployments), but what if you have multiple masters that also act as load balancers for themselves and no LB nodes? This would then cause an error. Could we instead check for the existence of lb_vm_group_size here in addition to checking its size?